### DISCUSSION TITLE
In wall 100v/220v electrical box option

### DISCUSSION THREAD

**Discussion by johnuopini** [FutureProofHomes/Satellite1-Hardware#8] [Q&A] [answered]
I have quite a few recessed electrical boxes wired at 220v, hole is quite standard (50mm diameter), since i guess most countries (at least US and EU) have a similar standard i think it would be great to have an option to either print and build an in-wall compatible case (with adapter) or have an already made version that basically acts as a smart in-wall sensor.

A bit like Sonoff NSPanel but without the screen, i think this could be very useful in a lot of contexts (ideally exposing also a button on the front this could become a killer smart "switch" for the home).

**Comment by that1guy** [comment]
Hi @johnuopini, thanks for sharing some thoughts here!

I want to validate my understanding of your idea - are you saying you're imagining a Sat1 voice assistant (containing mics, speakers, buttons, LEDs & sensors) being vertically mounted inside a wall at the approximate height of a standard light switch height and powered by mains (common residential 120V - 240V)?

**Comment by johnuopini** [comment]
Yes @that1guy thats correct, i mean either powered by mains or powered via 24v which is commonly used in passive PoE, important this is to be able to mount it directly into a recessed electrical box. There are quite a few companies providing similar proprietary solutions for hotels and residential, just as an example:

BTicino: https://catalogue.bticino.com/product/solution-for-residential-and-tertiary-building/living-now/living-now-my-home/BTI-KG8013-EN
Vimar: https://www.vimar.com/en/int/catalog/obsolete/index/code/30815.B
Echo Flex (not really in wall but the concept is similar): https://www.amazon.com/echo-flex/dp/b07mly3jkv

If voice is coupled with one or two buttons and a led this could be in my opinion a very useful replacement to standard in-wall switches or sensors.

**Comment by johnuopini** [comment]
For HA voice preview there is already some third party adapter like these (although they are very very thick and I think we could have better solutions):

https://www.printables.com/model/1114519-home-assistant-voice-recessed-outlet-mount

https://www.printables.com/model/1112597-home-assistant-voice-pe-minimalistic-socket-mount

And also a similar solution for Josh Nano:

https://seeless.com/collections/voice-control-mud-in-platforms/products/sl-jnai-62-josh-nano-mount

**Comment by that1guy** [comment]
Thanks for the clarification and details @johnuopini.  I completely get what you're asking for.

A product like this is not currently on our roadmap, but it is entirely possible to build a custom 3d enclosure around the Sat1 and to use something like this for power: https://www.amazon.com/PoE-Texas-Wall-USB-C-Adapter/dp/B0CP5MVK9D

**Reply by that1guy** [reply]
And you could route the Sat1 audio to in-ceiling speakers, or if you dared to try and put a speaker in a 4 gang box I guess nobody is stopping you.  Also, you can use the Sat1 buttons and HA automations to turn on lights for that room.  For example, double tap the action button and toggle the light on/off.

**Comment by johnuopini** [comment]
I have Sonos speakers all over the house as many probably also have. I think the main issue of using just an in wall USB-C adapter would be that the product would then be very thick off the wall.

Would be nice to have a very very thin version of the product with no speaker, just mics and sensors and use Sonos or any other HA player for output.

Anyway thanks for looking into this and thanks for point me to the adapter didn't find anything like that in EU stores.

**Comment by that1guy** [comment]
Thank for the idea @johnuopini.  I leave this discussion here to track the upvotes.

**Comment by Hedda** [comment]
> think the main issue of using just an in wall USB-C adapter would be that the product would then be very thick off the wall.

Tip is that there are many different exiisting in-wall USB-C (and USC-A) chargers already on the market; just search for "in-wall USB changer":

* https://www.google.com/search?q=in-wall+USB+changer

*

**Comment by that1guy** [comment, marked as answer]
We've decided not build build in-wall high voltage products due to regulation and challenges scaling products like this.   As we grow we might end up looking more into this, but not for now.
